tests/package-search-test.js

```diff
@@ -306,6 +298,160 @@ describeApplication('PackageSearch', () => {
     });
   });
 
+  describe('sorting packages', () => {
+    beforeEach(function () {
+      this.server.create('package', {
+        name: 'Academic ASAP'
+      });
+      this.server.create('package', {
+        name: 'Search Networks'
+      });
+      this.server.create('package', {
+        name: 'Non Matching'
+      });
+      this.server.create('package', {
+        name: 'Academic Search Elite'
+      });
+      this.server.create('package', {
+        name: 'Academic Search Premier'
+      });
+    });
+
+    describe('searching for packages', () => {
+      beforeEach(() => {
+        PackageSearchPage.search('academic search');
+      });
+
+      it('has search filters', () => {
+        expect(PackageSearchPage.$searchFilters).to.exist;
+      });
+
+      it('shows the default sort filter of relevance in the search form', () => {
+        expect(PackageSearchPage.getFilter('sort')).to.equal('relevance');
+      });
+
+      it("displays provider entries related to 'health analytics'", () => {
```

Comment:
Do we want to modify the test to say package instead of provider and also have the correct package name?

Reply:
Yes, thanks for catching those @sduvvuri-ebsco